lua/cmp_fuzzy_path/init.lua

```diff
@@ -74,7 +72,7 @@ source.stat = function(_, path)
 end
 
 local function find_pattern(cursor_before_line)
-  local match_start, match_end = COMPILED_PATH_REGEX:match_str(cursor_before_line)
```

Comment:
I would be better to not require a compilation of the path regex on every call.
Do not rely on nvim to cache common regexs for you.